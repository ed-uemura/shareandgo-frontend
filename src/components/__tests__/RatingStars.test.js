import {render, screen, cleanup} from '@testing-library/react'
import renderer from 'react-test-renderer'
import RatingSrars from '../RatingStars'


afterEach(() =>{
    cleanup();
});

test('matches snapshot', () =>{
    const rating  = '3';
    const tree = renderer.create(<RatingSrars rating={rating}/>).toJSON();
    expect(tree).toMatchSnapshot();
});

