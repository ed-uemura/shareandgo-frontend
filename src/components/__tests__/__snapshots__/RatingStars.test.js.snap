// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <i
    className="star"
  >
    <span
      className="material-symbols-outlined filled"
      id={0}
    >
      star
    </span>
  </i>
  <i
    className="star"
  >
    <span
      className="material-symbols-outlined filled"
      id={1}
    >
      star
    </span>
  </i>
  <i
    className="star"
  >
    <span
      className="material-symbols-outlined filled"
      id={2}
    >
      star
    </span>
  </i>
  <i
    className="star"
  >
    <span
      className="material-symbols-outlined"
      id={3}
    >
      star
    </span>
  </i>
  <i
    className="star"
  >
    <span
      className="material-symbols-outlined"
      id={4}
    >
      star
    </span>
  </i>
</div>
`;
